packages/core/src/errors/RequestError/index.ts

```diff
@@ -22,13 +22,16 @@ export default class RequestError extends Error {
   expose: boolean;
   data: unknown;
 
+  readonly #i18nInterpolation: Record<string, unknown>;
+
   constructor(input: RequestErrorMetadata | LogtoErrorCode, data?: unknown) {
     const {
       code,
       status = 400,
       expose = true,
       ...interpolation
     } = typeof input === 'string' ? { code: input } : input;
+
     const message = i18next.t<string, LogtoErrorI18nKey>(`errors:${code}`, {
```

Comment:
this is actually weird because message already has the i18n result

Reply:
Add some comments